docs/src/content/docs/it/guides/customization.mdx

```diff
@@ -0,0 +1,415 @@
+---
+title: Personalizzazione di Starlight
+description: Scopri come personalizzare il tuo sito Starlight con il tuo logo, caratteri personalizzati, design della pagina di destinazione e altro ancora.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import FileTree from '../../../../components/file-tree.astro';
+
+Starlight fornisce stili e funzionalità predefiniti sensati, così puoi iniziare rapidamente senza bisogno di configurazione.
+Se vuoi iniziare a personalizzare l'aspetto del tuo sito Starlight, questa guida fa al caso tuo.
+
+## Aggiungi il tuo logo
+
+L'aggiunta di un logo personalizzato all'intestazione del sito è un modo rapido per aggiungere il tuo marchio individuale a un sito Starlight.
+
+1. Aggiungi il file immagine del tuo logo alla directory `src/assets/`:
+
+    <FileTree>
+
+   - src/
+     - assets/
+       - **my-logo.svg**
+     - content/
+   - astro.config.mjs
+
+   </FileTree>
+
+2. Aggiungi il percorso del tuo logo come opzione [`logo.src`](/it/reference/configuration/#logo) di Starlight in `astro.config.mjs`:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Documentazione con il mio logo',
+   			logo: {
+   				src: './src/assets/my-logo.svg',
+   			},
+   		}),
+   	],
+   });
+   ```
+
+Per impostazione predefinita, il logo verrà visualizzato accanto al `title` del tuo sito.
+Se l'immagine del tuo logo include già il titolo del sito, puoi nascondere visivamente il testo del titolo impostando l'opzione `replacesTitle`.
+Il testo del `title` verrà comunque incluso per gli screen reader in modo che l'intestazione rimanga accessibile.
+
+```js
+starlight({
+  title: 'Documentazione con il mio logo',
+  logo: {
+    src: './src/assets/my-logo.svg',
+    replacesTitle: true,
+  },
+}),
+```
+
+### Varianti del logo chiaro e scuro
+
+Puoi visualizzare diverse versioni del tuo logo in modalità chiara e scura.
+
+1. Aggiungi un file immagine per ciascuna variante a `src/assets/`:
+
+   <FileTree>
+
+   - src/
+     - assets/
+       - **light-logo.svg**
+       - **dark-logo.svg**
+     - content/
+   - astro.config.mjs
+
+   </FileTree>
+
+2. Aggiungi il percorso alle varianti del tuo logo come opzioni "light" e "dark" invece di "src" in "astro.config.mjs":
+
+   ```js
+   starlight({
+     title: 'Documentazione con il mio logo',
+     logo: {
+       light: './src/assets/light-logo.svg',
+       dark: './src/assets/dark-logo.svg',
+     },
+   }),
+   ```
+
```

Comment:
Qua abbiamo aggiunto una nuova sezione in #637:

````md
## Enable sitemap

Starlight has built-in support for generating a sitemap. Enable sitemap generation by setting your URL as `site` in `astro.config.mjs`:

```js
// astro.config.mjs

export default defineConfig({
  site: 'https://stargazers.club/',
  integrations: [
    starlight({ title: 'Site with sitemap' }),
  ],
});
```
````

Potresti aggiungere questo nuovo paragrafo a questo PR?